catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_s2ls_intertidal_cyear_3.jsonnet

```diff
@@ -0,0 +1,135 @@
+local id = 'projects/geoscience-aus-cat/assets/ga_s2ls_intertidal_cyear_3';
+local subdir = 'geoscience-aus-cat';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local version = '1.0.0';
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  stac_version: ee_const.stac_version,
```

Comment:
could you set 'gee:status': 'beta' for a new dataset?

I need to make some changes in checks to avoid having to do this in the future, but for now this would simplify the workflow a bit for us.